bin/setup.deps

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+require 'pathname'
+require 'fileutils'
+
+# path to your application root.
+APP_ROOT = Pathname.new File.expand_path('../../',  __FILE__)
+
+
+def system!(*args)
+  system(*args) || abort("\n== Command #{args} failed ==")
+end
+
+def get_uid()
+  uid = `id -u`
+  uid.delete!("\n")
```

Comment:
Perhaps https://ruby-doc.org/core-2.4.5/String.html#method-i-strip is a better fit?